### DISCUSSION TITLE
How do I sponsor you?

### DISCUSSION THREAD

**Discussion by muukii** [swiftyfinch/Rugby#40] [Ideas]
Rugby is a super amazing tool. I hope this tool gets more improvements. but you need time to work this on. I'd like to know how do we support you.

**Comment by swiftyfinch** [comment]
@muukii Thank you for the suggestion!
I need time to compare available services, and I'll try to add one 👍🏻

**Comment by swiftyfinch** [comment]
@muukii Hello! I added information about sponsorship here #42
And again, thank you for the suggestion)
